Source/Resources/Content/MSGraph/Groups/BUILTIN - Autopilot Devices - User Driven.json

```diff
@@ -1,6 +1,6 @@
 {
   "assignedLicenses": [],
-  "description": "Contains all corporate devices, except those explicitly tagged as 'Autopilot-SelfDeploying'. Not yet enrolled devices eligible for autopilot are identified as those having a 'ZTDId' marker.",
+  "description": "What is it: A dynamic device group that is used for assigning the User-Driven Intune Autopilot profile to devices. \n Simeon Notes: Includes all devices that are not in the Azure AD group, 'BUILTIN - Autopilot Devices - Self Deploying'. \n Learn More: https://docs.microsoft.com/en-us/mem/autopilot/user-driven",
```

Comment:
**What does this do?**
Creates a dynamic device group for assigning the user-driven Autopilot profiles to devices. This group includes all devices that are not in the Azure AD group "Autopilot Devices - Self Deploying."

**Why should you use this?**
If you want to identify a group of users that can assign user-driven Intune Autopilot profiles outside of the self deploying group.  

**What is the end user impact?**
Users of this group will be able to assign Intune Autopilot profiles to devices outside of those that self deploy.

**Learn more**
NA